python-sgd-regression/sklearn_to_pfa/sklearn_to_pfa.py

```diff
@@ -0,0 +1,59 @@
+# TODO: move to a separate repository or at least python-mip
```

Comment:
This is very, very rough basis for library for converting scikit-learn models into PrettyPFA. Again, I'm not sure where to put it - should I put it into `python-mip` or maybe create a new repo under `HBPMedical` (in that case, could you please tell me how are you using atomist for creating python packages?). Thanks!